src/conn/conn_api.c

```diff
@@ -1149,6 +1150,13 @@ __conn_reconfigure(WT_CONNECTION *wt_conn, const char *config)
 	__wt_free(session, conn->cfg);
 	conn->cfg = p;
 
+	/*
+	 * Fourth, run any diagnostic commands that were included. Do that last,
+	 * reconfiguration might include other configuration changes to support
+	 * a command.
+	 */
+	WT_ERR(__conn_verbose_cmd(session, config));
```

Comment:
> and avoiding thread disruption on verbose-only is a good idea.

To be clear, there shouldn't be any thread disruption, thread disruption would be evidence of a bug in the code.  That said...